polytracker/src/taintdag/polytracker.cpp

```diff
@@ -7,212 +7,165 @@
  */
 
 #include "taintdag/polytracker.h"
-
+#include "taintdag/util.h"
 #include <sanitizer/dfsan_interface.h>
 
 #include <sys/stat.h>
 
-#include "taintdag/error.hpp"
+#include "taintdag/error.h"
 #include "taintdag/fnmapping.h"
 #include "taintdag/fntrace.h"
 
 namespace fs = std::filesystem;
```

Comment:
We should probably remove or make this more local.

Reply:
I'll fix!

Reply:
Done.